build/webvr-polyfill.js

```diff
@@ -2489,7 +2497,7 @@ CardboardVRDisplay.prototype.beginPresent_ = function() {
     this.exitPresent();
   }.bind(this));
 
-  if (Util.isLandscapeMode() && Util.isMobile()) {
+  if (this.rotateInstructions_ &&Util.isLandscapeMode() && Util.isMobile()) {
```

Comment:
Consider using a linter. Looking for syntax nits is a bad use of human resources. Let's use machines until they rebel against us and demand rights.